rotkehlchen/globaldb/upgrades/manager.py

```diff
@@ -73,6 +74,10 @@
         from_version=9,
         function=migrate_to_v10,
     ),
+    UpgradeRecord(
+        from_version=10,
+        function=migrate_to_v11,
+    ),
```

Comment:
How does introducing a new version affect to the thing you did for the ranges in the updates before the globaldb start?

Reply:
it does not affect it. since there's no breaking change in the assets schema.